src/Adapter/State/CommandHandler/AddStateHandler.php

```diff
@@ -39,7 +38,7 @@
 /**
  * Handles creation of state
  */
-final class AddStateHandler extends AbstractStateHandler implements AddStateHandlerInterface
```

Comment:
Do not remove this. It will be used in later stages in development where adapters will be removed

Reply:
Ok, will revert it back. Have you seen review comment above about AddStateHandlerInterface?

Reply:
The review comment refers to a php comment above the class

Reply:
done Raimondas - we have it again.